__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+afterAll(() => db.end());
+beforeEach(() => seed(testData));
+
+describe("GET api/categories", () => {
+  test("200: should succesfully get categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: categories }) => {
+        categories.forEach((category) => {
+          expect(category).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+        expect(categories.length).toBe(4)
+      });
+  });
+});
```

Comment:
We should always test for any error handling that you've included. Add a test for the 404 error handling in your app. 😄

Reply:
Will do thank you John.